frontend/public/components/overview/index.tsx

```diff
@@ -447,42 +502,23 @@ class OverviewMainContent_ extends SafetyFirst<OverviewMainContentProps, Overvie
     });
   }
 
-  groupItems(items: OverviewItem[], label: string): OverviewGroup[] {
-    const compareGroups = (a, b) => {
-      if (a.name === EMPTY_GROUP_LABEL) {
-        return 1;
-      }
-      if (b.name === EMPTY_GROUP_LABEL) {
-        return -1;
-      }
-      return a.name.localeCompare(b.name);
+  getGroupOptionsFromLabels(items: OverviewItem[]): any {
```

Comment:
Since this returns group options that aren't necessarily derived from the resource labels, we can probably just rename this.

```suggestion
  getGroupOptions(items: OverviewItem[]): any {
```